pyproject.toml

```diff
@@ -27,7 +27,6 @@ sdist.include = ["src/jax_finufft/jax_finufft_version.py"]
 wheel.install-dir = "jax_finufft"
 minimum-version = "0.5"
 build-dir = "build/{wheel_tag}"
-wheel.py-api = "cp312"
```

Comment:
Did we not want to build Limited API wheels anymore?

Reply:
I couldn't get the `c12-abi2` wheels to work properly on my installation of 3.13. Perhaps there's something to do, but killing this made us build a proper 3.13 wheel that works. Seems fine for now!